new/cookbook_quality_metrics.md

```diff
@@ -0,0 +1,48 @@
+---
+RFC: unassigned
+Author: Nathen Harvey <nharvey@chef.io>
+Status: Draft
+Type: Informational
+<Replaces: RFCxxx>
+<Tracking:>
+<  - https://github.com/chef/chef/issues/X>
+---
+
+# Cookbook Quality Metrics
+
+The question of what makes a good cookbook has been asked since the dawn of Chef.  As a community we've struggled with answering this in part because each recipe is unique to the specific requirements it is trying to meet.  Allowing Supermarket users to rate cookbooks is one thing that we've tried in the past.  This did not work because you simply do not have enough knowledge about the quality of a cookbook when you discover and download it from the Supermarket.  Similarly, download counts cannot be trusted because of [the way some older versions of the berkshelf-api server work](https://www.chef.io/blog/2015/01/21/those-pesky-supermarket-download-counts/).
+
+The Supermarket should make it easy for anyone to find quality cookbooks.  In order to do so, we must agree on some qualities that indicate whether or not a cookbook is considered to have high quality.  Ideally, these qualities are objective and able to be determined automatically.  These qualities should be defined and agreed on by the community.
+
+## Motivation
+
+    As a cookbook author,
+    I want to write cookbooks that can be shared with and easily used by the community
+    so that it is easier to get going with Chef.
+
+    As a Supermarket user,
+    I want to identify the relative quality of a given cookbook,
+    so that I can spend less time looking for a good cookbook and more time automating.
+
+## Specification
+
+A [Cookbook Quality Metrics](https://github.com/chef-cookbooks/cookbook-quality-metrics) system will be implemented to provide a quality score for each cookbook.  This score will be visible on the Supermarket.
+
+* The community will be able to collaborate on the metrics.
+* Each quality metric will add or remove points from a cookbook's score.
```

Comment:
I agree that specific scores is probably too much. I think I'd most like a bunch of true/false metrics, like the ones already in that repo, and then maybe:

Green - all metrics passed - this looks good
Yellow - 1 or more metrics did not pass - proceed with caution
Red - 3+ metrics did not pass - you should probably avoid this